src/usr/share/netcheck/netcheck.py

```diff
@@ -776,3 +794,55 @@ def _calculate_next_log_time(self, loop_time):
         Returns the datetime that the logging should occur.
         """
         return loop_time + datetime.timedelta(seconds=self.config['periodic_status_delay'])
+
+    def _check_for_gateway_change(self):
+        """Checks the current state of the default gateway and issues a broadcast if it is
+        different from the prior state.
+        """
+
+        new_gateway_state = self._get_default_gateway_state()
```

Comment:
I'd call this new_default_gateway_state.

Reply:
Actually, to be consistent with other parts of this code, I'd probably call this default_gateway_state and the instance variable prior_default_gateway_state.